lib/router.js

```diff
@@ -54,7 +54,7 @@ app.disable("etag");
 
 var requestId = 0;
 app.route("*").all(function(req, res, next) {
-  debug.requestCounter(requestId++, req.url);
+  debug.requestCounter(requestId++, req.method, req.url);
```

Comment:
I've added the http verb to the request debugging, making it easier to differentiate between GET and PATCH.